### ISSUE TITLE
2020-9-14

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#314]
691.3.convert it to the following unconstrained loss by means of soft constraints; (2020-9-7)
692.4.adopt stochastic gradient descent (SGD) to minimize the above loss function; (2020-9-7)
693.Reducing False Negative Labels
694.1.randomly sampling may introduce many false negative labels into training; (2020-9-7)
695.2.set different probabilities for replacing the head or tail entity when corrupting the triplet, depends on one-to-many, many-to-one or many-to-many; give more chance to replacing the head entity if the relation is one-to-many and give more chance to replacing the tail entity if the relation is many-to-one; (2020-9-7)
696.Experiments
697.1.link prediction, triplets classification, relational fact extraction; (2020-9-7)
698.Link Prediction
699.1.Rather than requiring one best answer, this task emphasizes more on ranking a set of candidate entities from the knowledge graph; (2020-9-7)
